scripts/lossPersistence/pp.yml

```diff
@@ -0,0 +1,70 @@
+script: pp.R
+description: "This sample script shows how it works."
+external_link: https://github.com/GEO-BON/biab-2.0
+inputs:
+  WKT_area:
+    label: WKT_area
+    description: WKT_area
+    type: text
+    example: 'MULTIPOLYGON (((-74.51422355910061 6.287998672327671,-74.28889444279572 6.249776453227831,-74.33011318358322 6.072279835991389,-74.29439027490073 6.066817465067729,-74.24767570200825 5.982143668342713,-74.24767570200825 5.8919926391689925,-74.48124856647063 5.867403444598711,-74.53620688752059 5.973948731032553,-74.69283810251301 5.982143668342713,-74.59940895672806 6.085935520826565,-74.51422355910061 6.287998672327671)))'
+  collection_path:
+    label: dir_collection
+    description: dir_collection
+    type: text
+    example: '/scripts/lossPersistence/input/Colombia_pp_collection'
+  epsg:
+    label: epsg
+    description: epsg
+    type: int
+    example: 4326
+  resolution:
+    label: resolution
+    description: resolution
+    type: int
+    example: 1000
+  time_period:
+    label: time_period

```

Comment:
All labels should be human-readable format. Right now they look like stake case, a duplicate of the key.